### DISCUSSION TITLE
esp01 onboard led

### DISCUSSION THREAD

**Discussion by onkel-joerg** [arendst/Tasmota#11464] [Q&A] [answered]
Hello
here is a noob question.
I have an ESP01 with an DHT11 set bought at aliexpress
Flashed with Tasmota works like a charm. GPIO Pin 2 set to DHT11 - everything is fine.
How do I switch off the onboard LED?
Yes I know, one could solder it off the board but I would like to keep it. Just switch it on/off.
Probably I'm just too stupid to search and it's explained foolproof somewhere here.

**Comment by barbudor** [comment, marked as answer]
To control a led, as you have no relay on the board you can either :
- Define the GPIO as a Relay and use commands POWER ON, POWER OFF to control. And use PowerOnState command to define the default 
- Define the GPIO as a Led and use command LedPower in a similar way

See : https://tasmota.github.io/docs/Commands/

You need to identify the proper GPIO and for this Google is your friend (or your multimeter in continuity tester mode)
However some ESP01 have a red led directly connected to the power and cannot be controlled by software. In that case, heating up your soldering iron the only way.

Enjoy

**Reply by Milbert-R** [reply]
On my ESP-01 the blue LED is corresponding with GPIO02...
![esp01_led](https://user-images.githubusercontent.com/37844838/112523464-12396580-8d9f-11eb-8663-800d702401d5.png)

**Reply by onkel-joerg** [reply]
Sorry for incorrect question.
I meant the red LED.


Thank you for your help

**Reply by ascillato** [reply]
The red led can not be controlled. Is connected directly to 3.3v

**Comment by Jason2866** [comment]
I never saw a ESP-01 where the red onboard led is not connected to VCC.
